src/components/cylc/tree/TreeItem.vue

```diff
@@ -14,15 +14,15 @@
       >{{ isExpanded ? '&#9661;' : '&#9655;' }}</v-flex>
       <!-- the node value -->
       <!-- TODO: revisit these values that can be replaced by constants later (and in other components too). -->
-      <div class="node-data" @click="nodeClicked" v-if="node.type === 'cyclepoint'">
+      <div :class="getNodeDataClass()" @click="nodeClicked" v-if="node.type === 'cyclepoint'">
```

Comment:
Adding the `node.type` as a class, so that tests can easily find `div.node-data` for all the nodes with data or, something like `div.cyclepoint-node-data` for just the cycle point node data.